mysql_scripts/table_creation/FullDB.sql

```diff
@@ -189,3 +190,14 @@ CREATE TABLE UserPermission (
 	FOREIGN KEY(permissionId) REFERENCES Permission(permissionId),
 	CONSTRAINT UNIQUE unique_permission (userId, permissionId)
 );
+
+CREATE TABLE VirtualAppointmentConsent (
+	virtualAppointmentConsentId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+	createdAt DATETIME NOT NULL DEFAULT NOW(),
+	reviewConsent BOOLEAN NOT NULL,
+	virtualConsent BOOLEAN NOT NULL, /* virtual is a keyword */
+    signature VARCHAR(355) NOT NULL,
+    appointmentId INTEGER UNSIGNED NOT NULL,
+    UNIQUE (appointmentId),
```

Comment:
nit: It's nice to have a defined constraint name so that we can easily delete it in the future if necessary. If we don't give it a name, the DB assigns it some random name that's harder to find
```suggestion
    CONSTRAINT UNIQUE unique_appointment_id (appointmentId),
```

Reply:
Changed